src/App.tsx

```diff
@@ -46,6 +47,7 @@ class App extends Component<void, AppState> {
     this.pauseCurrentPlayback = this.pauseCurrentPlayback.bind(this);
     this.resumeCurrentPlayback = this.resumeCurrentPlayback.bind(this);
     this.loadDevices = this.loadDevices.bind(this);
+    this.goBack = this.goBack.bind(this);
```

Comment:
Random note, I _think_ in newer versions of React there's a way to set this up without all the annoying binding? I'll have to look into it

Reply:
you talking about function components (aka react hooks) or smth different?